src/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { memo } from 'react';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todo: Todo,
+  onDeleteTodo: (id: number) => void,
+  hasLoadingStatus: boolean,
+}
+
+export const TodoInfo: React.FC<Props> = memo(({
+  todo,
+  onDeleteTodo,
+  hasLoadingStatus,
+}) => {
+  const { title, completed, id } = todo;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={cn('todo', {
+        completed: completed === true,
```

Comment:
```suggestion
        completed,
```
You don't need to compare with true as the value is already boolean, and you can simplify